tests/test_skills/test_abstract_round_abci/test_base.py

```diff
@@ -1169,8 +1169,10 @@ def setup(self) -> None:
     @pytest.mark.parametrize("flag", (True, False))
     def test_is_abstract(self, flag: bool) -> None:
         """Test `is_abstract` property."""
-        self.abci_app._is_abstract = flag
-        assert self.abci_app.is_abstract is flag
+        # copy the class to ensure there are no side effects
+        CopyOfAbciApp = type('AbciApp', AbciApp.__bases__, dict(AbciApp.__dict__))
+        CopyOfAbciApp._is_abstract = flag
+        assert CopyOfAbciApp.is_abstract() is flag
```

Comment:
to prevent unwanted side effects, because

```
>>> AbciApp.is_abstract()
True
... a = copy.deepcopy(AbciApp)
... a._is_abstract = 1234
>>> AbciApp.is_abstract()
1234